qa/rpc-tests/fundrawtransaction.py

```diff
@@ -32,8 +32,8 @@ def setup_network(self, split=False):
         self.sync_all()
 
     def run_test(self):
-        print "Mining blocks..."
-        feeTolerance = Decimal(0.00000002) #if the fee's positive delta is higher than this value tests will fail, neg. delta always fail the tests
+        print("Mining blocks...")
+        feeTolerance = Decimal("0.00000002") #if the fee's positive delta is higher than this value tests will fail, neg. delta always fail the tests
```

Comment:
@Eirik0 this is my favorite bug today.  `Decimal(SOMEFLOAT)` creates a Decimal with the same number of sigfigs as the original float (69 base 10 digits in this case), beyond the 10^-8.   So then there's:
69 digits beyond the 10^-8
8 digits from for the first 8
That's 77 digits total.
10^77 is slightly less than 2^256
My guess is that the float has as many digits as can fit into 256 bits.
Interestingly, the Decimal representation of `0.00000002`  <- float has an `E-8` suffix.

Reply:
Hehh..
`Decimal(5.0)` -> `Decimal('5.0')`
`Decimal(5.1) -> Decimal('5.0999999999999996447286321199499070644378662109375')`